db-engine/src/main/java/com/splicemachine/db/impl/jdbc/TransactionResourceImpl.java

```diff
@@ -173,7 +172,9 @@
         drdaID = info.getProperty(Attribute.DRDAID_ATTR, null);
         rdbIntTkn = info.getProperty(Attribute.RDBINTTKN_ATTR, null);
         ipAddress = info.getProperty(Property.IP_ADDRESS, null);
-        defaultSchema = info.getProperty(Property.CONNECTION_SCHEMA, null);
+        defaultSchema = info.getProperty(Property.CONNECTION_CURRENT_SCHEMA, null);
+        if (defaultSchema == null)
+            defaultSchema = info.getProperty(Property.CONNECTION_SCHEMA, null);
```

Comment:
It seems we support both `schema` and `currentSchema` then and `currentSchema` has priority in a sense that if it's set, then `schema` is ignored. Is that intended?

Reply:
e.g. what's the expected behavior of `schema=A;currentSchema=B`.

Reply:
Yes, @ascend1 and @hatyo , the code above is intended, so for the scenario @hatyo raised, we have a deterministic behavior. Basically, currentSchema will take precedence over schema.

Reply:
Thanks!